### DISCUSSION TITLE
How do I get all menu items activated?

### DISCUSSION THREAD

**Discussion by bernardroth** [iiab/iiab#2969] [Q&A] [answered]
After I install the largest option I install the medical version and try to add the other menu items, but they do not work. Only the medical and map options work. The school options fail. When I click on Wikipedia I get "Undefined Zim in a Menu Definition". Radiolab "404 Not Found".

Is there a way I can get every option (medical and education) to work at the same time?

**Comment by tim-moody** [comment]
>After I install the largest option I install the medical version and try to add the other menu items

I take it the largest option was part of the iiab script. How did you do the medical version?

Adding menu items does not add content. Rather adding content adds menu items.

**Reply by bernardroth** [reply]
I used the script and selected the largest option (education). Then I used the admin page and installed the medical. After the reboot it only shows the medical menu items so I go to admin content menus and try adding the education items back.

**Comment by tim-moody** [comment, marked as answer]
OK. That all makes sense, but unfortunately there are a couple of things.

First, installing the large option only installs, services, not content. Second, presets (Quick Pick) are only expected to be installed once so the menu is overwritten.

At this point you have medical content and manually created menu items for both medical and education (you have not mentioned language). You still need to install all the education content.

I would go back to Quick Pick (presets) and take the school option that matches the language you want. This will replace your home page menu, but it will install all the content. (make sure your sd card is at least 128G)

You can then go to Content Menu and take Update Home Menu. This should more or less get you what you want.

**Reply by bernardroth** [reply]
Thanks. That is what I was missing. It's a two step install.

I wish there was one page for installing all of the additional content.

**Comment by tim-moody** [comment]
>I wish there was one page for installing all of the additional content.

I have been planning that but not done it yet. https://github.com/iiab/iiab-admin-console/issues/427

Note that you could still have added menu items for which you have not gotten content. Either download that content via Install Content or remove the menu items under Edit Menus

**Comment by holta** [comment]
Many thanks to @bernardroth and @tim-moody who are currently following up exactly 6 months later — adding improvements here:

- #3124
